### PR TITLE
VRO Console: monitor Camel messages

### PR DESCRIPTION
**Pull request by yoomlam**
## What was the problem?

As a follow-on to #531, we'd like monitor claims as they are being processed along Camel routes for diagnosing problems particularly in prod.
The RabbitMQ Management UI is not sufficient.

Associated tickets or Slack threads:

- #562 

## How does this fix it?

Adds [Camel wiretaps](https://camel.apache.org/components/3.18.x/eips/wireTap-eip.html) at critical points in Camel routes to monitor messages and their content.

## How to test this PR
- `export COMPOSE_PROFILES=debug` -- the console container only starts if the docker-compose `debug` profile is enabled.
- Start VRO. Note the `vro-console-1` container; check out the logs: `docker logs vro-console-1`.
- Attach to the container: `docker attach vro-console-1`
- On the `groovy:000>` prompt, try the following:
```groovy
// Check out http://localhost:15672/#/queues for current queues and see how `console-*` queues are added as the following commands are run.

// Initially no routes in the CamelContext of the Console container
groovy:000> camel.routes
===> []

groovy:000> wireTap claim-submitted
// Now, submit a claim using Swagger
// Expect to see a log message

groovy:000> wireTap generate-pdf
// Now, generate a pdf using Swagger
// Expect to see a log message

groovy:000> camel.routes
===> [
  Route[rabbitmq://tap-generate-pdf?exchangeType=topic&queue=console-generate-pdf -> null], 
  Route[rabbitmq://tap-claim-submitted?exchangeType=topic&queue=console-claim-submitted -> null]
]
```
- Stopping the Console container will remove the `console-*` queues.
- Check that the `app` container logs has no errors.

### CODE REVIEW

File: console/src/main/groovy/gov/va/vro/consolegroovy/VroConsoleShell.groovy [hunk 1]
```diff
@@ -33,16 +34,17 @@ class VroConsoleShell {
   VeteranRepository veteranRepository
 
   String submitSeda() {
-    return producerTemplate.requestBody("seda:foo", "Hello", String.class)
+    return producerTemplate.requestBody("seda:foo", "Hello", String)
   }
```
Comment (yoomlam) [inline console/src/main/groovy/gov/va/vro/consolegroovy/VroConsoleShell.groovy:38]: Placeholder for future capability to inject messages into Camel routes.

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java [hunk 2]
```diff
@@ -64,9 +64,18 @@ private void configureRouteClaimSubmit() {
         // Use Properties not Headers
         // https://examples.javacodegeeks.com/apache-camel-headers-vs-properties-example/
         .setProperty("diagnosticCode", simple("${body.diagnosticCode}"))
+        .wireTap(wireTapTopicFor("claim-submitted"))
```
Comment (yoomlam) [inline service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java:67]: Declare some wireTap points for the Console to subscribe to.

**Review by au2 — APPROVED** [review]
Works as described. Did not see anything in the code. Maybe do log.info instead of log.warn for the Execute args. Warning indicates some type of error to me. Also looks like log messages is 1000 characters long and gets clipped for longer. Not sure if there is a setting in the console for it.

**Comment by yoomlam** [comment]
@au2 Hmmm. Maybe I'll also save the JSON to a file (once we are allowed to store PII), using a timestamp and claim id as the filename.

**Review by sethdarragile6 — APPROVED** [review]
LGTM (looks groovy to me)

**Pull request merged**